src/drivers/driver.ts

```diff
@@ -780,11 +759,11 @@ export abstract class CMakeDriver implements vscode.Disposable {
         const child = this.executeCommand(program, args, undefined, { silent: true });
         try {
             const result = await child.result;
-            log.debug(localize('command.version.test.return.code', 'Command version test return code {0}', nullableValueToString(result.retc)));
+            log.trace(localize('command.version.test.return.code', '"{0}" returned code {1}', `${program} --version`, nullableValueToString(result.retc)));
```

Comment:
should `--version` be replaced with `${args}`?

Reply:
It's currently always just `--version` with one exception where it's `/?`, but I'll fix it.